### ISSUE TITLE
Insert button changes changes shape on click

### ISSUE THREAD

**Issue by mvdbeek** [galaxyproject/galaxy#19361]
[Release Notes ](https://docs.galaxyproject.org/en/latest/releases/index.html)
This Galaxy server version is 25.0.dev0, and the web client was built on Wednesday Dec 4th 10:32:30 2024 GMT-8 .

Only in MAC/Safari , Chrome and Firefox are Ok. 
1. Clicking on "insert" make the button scrunch up. This is true even if the window is expanded to be "wide" when I click. 
2. Opening that panel really wide then going narrow again resize correctly.
3. Clicking on "copy" also resets to the right size.
4. I see that removing these excess published workflow is planned for 24.2. Maybe this is minor since these won't be listed at all soon. 

![Screen Shot 2024-12-19 at 1 26 47 PM](https://github.com/user-attachments/assets/5ef86d72-8cf7-4c1b-9123-e729f9552331)

_Originally posted by @jennaj in https://github.com/galaxyproject/galaxy/issues/19265#issuecomment-2555819557_

**Comment by itisAliRH** [comment]
@mvdbeek I couldn't reproduce this issue on Safari 18.2. The insert button for shared workflows is not working now and the import button should not be there either (no need for that). But the shared workflows are dropped from this activity in #19312

**Comment by itisAliRH** [comment]
@mvdbeek Yes. The import button should not be there too as it is not relevant to the editor functionality. Only the copy is working for not-owned public workflows and can be there. Should I push a temporary fix in 24.2?

**Comment by mvdbeek** [comment]
Sorry, what I meant is that there is no import button in the editor, but can you check that where the button is actually visible it behaves correctly ?
